--- origsrc/gnome-menus-2.22.2/libmenu/canonicalize.c	2008-05-26 11:10:56.000000000 -0500
+++ src/gnome-menus-2.22.2/libmenu/canonicalize.c	2008-05-29 23:00:25.000000000 -0500
@@ -32,6 +32,10 @@
 #include <errno.h>
 #include <stddef.h>
 
+#ifndef MAXSYMLINKS
+#  define MAXSYMLINKS 32
+#endif
+
 /* Return the canonical absolute name of file NAME.  A canonical name
    does not contain any `.', `..' components nor any repeated path
    separators ('/') or symlinks.  All path components must exist.  If
