--- origsrc/gnome-menus-2.30.5/python/Makefile.am	2010-09-20 10:44:39.000000000 -0500
+++ src/gnome-menus-2.30.5/python/Makefile.am	2010-12-13 21:12:30.050866500 -0600
@@ -14,7 +14,7 @@ gmenudir = $(pyexecdir)
 gmenu_LTLIBRARIES = gmenu.la
 
 gmenu_la_SOURCES = gmenu.c
-gmenu_la_LDFLAGS = -module -avoid-version -fPIC -export-symbols-regex initgmenu
+gmenu_la_LDFLAGS = -module -avoid-version -no-undefined -export-symbols-regex initgmenu
 gmenu_la_LIBADD = $(GLIB_LIBS) $(PYTHON_LIBS) $(top_builddir)/libmenu/libgnome-menu.la
 
 -include $(top_srcdir)/git.mk
