--- origsrc/gnome-menus-2.22.2/libmenu/canonicalize.c	2008-05-26 11:10:56.000000000 -0500
+++ src/gnome-menus-2.22.2/libmenu/canonicalize.c	2008-05-29 23:00:25.000000000 -0500
@@ -32,6 +32,10 @@
 #include <errno.h>
 #include <stddef.h>
 
+#ifndef MAXSYMLINKS
+#  define MAXSYMLINKS 32
+#endif
+
 /* Return the canonical absolute name of file NAME.  A canonical name
    does not contain any `.', `..' components nor any repeated path
    separators ('/') or symlinks.  All path components must exist.  If
--- origsrc/gnome-menus-2.22.2/python/Makefile.am	2008-05-26 11:10:53.000000000 -0500
+++ src/gnome-menus-2.22.2/python/Makefile.am	2008-05-29 23:00:25.015625000 -0500
@@ -1,5 +1,7 @@
 NULL =
 
+PYTHON_LIBS = -L$(prefix)/lib/python$(PYTHON_VERSION)/config -lpython$(PYTHON_VERSION)
+
 INCLUDES =				\
 	-DGMENU_I_KNOW_THIS_IS_UNSTABLE	\
 	-I$(srcdir)/../libmenu		\
@@ -14,6 +16,6 @@
 gmenudir = $(pyexecdir)
 gmenu_LTLIBRARIES = gmenu.la
 
-gmenu_la_LDFLAGS = -module -avoid-version -fPIC -export-symbols-regex initgmenu
-gmenu_la_LIBADD  = $(GLIB_LIBS) $(top_builddir)/libmenu/libgnome-menu.la
+gmenu_la_LDFLAGS = -module -avoid-version -no-undefined -export-symbols-regex initgmenu
+gmenu_la_LIBADD  = $(GLIB_LIBS) $(top_builddir)/libmenu/libgnome-menu.la $(PYTHON_LIBS)
 gmenu_la_SOURCES = gmenu.c
